extensions-core/kinesis-indexing-service/src/main/java/org/apache/druid/indexing/kinesis/supervisor/KinesisSupervisorIOConfig.java

```diff
@@ -85,8 +87,16 @@ public KinesisSupervisorIOConfig(
         completionTimeout,
         lateMessageRejectionPeriod,
         earlyMessageRejectionPeriod,
+        null,
```

Comment:
We can just pass `autoscalerConfig` here and it will be null because if its not null then exception will be thrown at line 96.

Reply:
changed.